packages/base/room.gts

```diff
@@ -251,14 +251,16 @@ class EmbeddedMessageCard extends Component<typeof MessageCard> {
   });
 }
 
-class MessageCard extends FieldDef {
+
+export class MessageCard extends FieldDef {
   @field author = contains(RoomMemberCard);
   @field message = contains(MarkdownField);
   @field formattedMessage = contains(StringField);
   @field created = contains(DateTimeCard);
   @field attachedCardId = contains(StringField);
   @field index = contains(NumberField);
   @field transactionId = contains(StringField);
+  @field command = contains(StringField);
```

Comment:
Using a string to store this doesn't feel great - I'd like something more structured but patching inherently has an unknown structure up-front. Does this need to be done differently for getting this merged in?